packages/salesforce/src/Adaptor.js

```diff
@@ -626,19 +739,17 @@ export function toUTF8(input) {
 }
 
 /**
- * Send a HTTP request using connected session information.
- *
- * @example
- * request('/actions/custom/flow/POC_OpenFN_Test_Flow', {
- *   method: 'POST',
+ * Send a request to salesforce server configured in `state.configuration`.
+ * @public
+ * @example <caption>Make a POST request to a custom Salesforce flow</caption>
+ * request("/actions/custom/flow/POC_OpenFN_Test_Flow", {
+ *   method: "POST",
  *   json: { inputs: [{}] },
  * });
- * @param {string} url - Relative to request from
- * @param {object} options - The options for the request.
- * @param {string} [options.method=GET] - HTTP method to use. Defaults to GET
- * @param {object} [options.headers] - Object of request headers
- * @param {object} [options.json] - A JSON object to send as the request body.
- * @param {string} [options.body] - HTTP body (in POST/PUT/PATCH methods)
+ * @function
+ * @param {string} path - The Salesforce API endpoint.
+ * @param {FullRequestOptions} [options] - Configure headers, query and body parameters for the request.
+ * @state {SalesforceState}
```

Comment:
For `request`, `get` and `post`, can we add a better  description of the returned state? Is it like `{ response, body } ?

Reply:
@josephjclark it depends with the request endpoint response, Some returns an object some returns an array and the contents of the response is different for each endpoint. 

For example 
`get('/services/data')` returns an array of objects with value like `{label, url, version` while `get('/sobject/Account')` returns and object with properties like `{objectDescribe: {}, url: {}, recentItems: []}`

I have updated it to 
```
* @property data - API response data. Can be either an object or array of objects.
```
How does that sound ?